Save fastboot snapshot when exiting validator (#9713)
anza-xyz/agave: CHANGELOG.md

  * `--wait-for-exit` (`exit` subcommand)
#### Deprecations
* Using `mmap` for `--accounts-db-access-storages-method` is now deprecated.

> #### Changes

Reviewer: We need this `### CLI` tag - it is referring to another section:
<img width="1107" height="952" alt="image" src="https://github.com/user-attachments/assets/75f12211-d821-464a-a13b-f83e32b06d01" />

> * `agave-validator exit` now saves bank state before exiting. This enables restarts from local state when other snapshots are disabled.

Reviewer 2: How about this:

```suggestion
* `agave-validator exit` now saves bank state before exiting. This enables restarts from local state when snapshot generation disabled.
```

We use the wording "snapshot generation" in the `--help`, so hopefully that connects them more.

> * `agave-validator exit` now saves bank state before exiting. This enables restarts from local state when snapshot generation disabled.

Author: Thanks! Wording is definitely better.
